docs/source/embedded-backends.md

```diff
@@ -0,0 +1,24 @@
+(embedded-backends)=
+# Backends
+
+Available hardware acceleration backends for embedded systems.
+
+## DSP Acceleration
+
+- {doc}`embedded-cadence` — Cadence Xtensa DSP processors
+
+## NPU Acceleration
+
+- {doc}`embedded-arm-ethos-u` — ARM Ethos-U NPU acceleration
+- {doc}`embedded-nxp` — NXP eIQ Neutron Backend
+
+## Custom Development
+
+- {doc}`backend-delegates-integration` — Custom backend development guide
+
+```{toctree}
+:hidden:
+embedded-cadence
+embedded-arm-ethos-u
+embedded-nxp
+backend-delegates-integration
```

Comment:
backend-delegates-integration in embedded doesn't make sense. bring to "Backend Overview" section